WooCommerce/Classes/POS/Models/CartItem.swift

```diff
@@ -11,4 +11,8 @@ struct CartItem {
         self.item = item
         self.quantity = quantity
     }
+
+    public static func == (lhs: CartItem, rhs: CartItem) -> Bool {
+        lhs.item.productID == rhs.item.productID
```

Comment:
Is this the right equatable property to use here? 2 equal products would have the same `productID` but not be the same item, so comparing itemID's might make more sense? 🤔